course7/homeworks/patrickh/complex/complex_linked_list_node.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "complex.h"
+#include "memory"
+struct ComplexLinkedListNode
```

Comment:
This is a private class, an implementation detail. It doesn't deserve to be a top-level class, not to mention it's own source file. IMO this should be a private nested class of the list.